spec/requests/api/articles/index_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "GET /api/articles", type: :request do
```

Comment:
Single quotes or double quotes but not both

Reply:
WIP! Correction coming soon.

I found some things to correct and divided them into two groups - two commits.
But since the first correction was not accepted, I did not report the second.
Because I understood that I didn't need to change the codes of other team members.

My formatter uses the ' - so I accepted that too.